packages/core/lib/console.js

```diff
@@ -243,10 +243,7 @@ class Console extends EventEmitter {
   provision() {
     let files;
     try {
-      const unfilteredFiles = fse.readdirSync(
-        this.options.contracts_build_directory
-      );
-      files = unfilteredFiles.filter(file => file.endsWith(".json"));
+      files = fse.readdirSync(this.options.contracts_build_directory);
```

Comment:
I'm not sure it is good to remove that filter. I think it was added at some point to prevent errors when there were other things in the `build` folder.

Reply:
Good catch! I'll revert this change.

Reply:
Actually, I think we can do the filtering in one loop